### PR TITLE
Support scan in storage

### PR DESCRIPTION
**Pull request by critical27**
Based on https://github.com/vesoft-inc/nebula-common/pull/246. Support scan data from storage within specified partition. Almost same as what we did in 1.0, except that we don't need to return the encoded value, just return the decoded value instead.

Could just review the new code, the existing code has expired which is from 1.0.

### CODE REVIEW

File: src/storage/exec/QueryUtils.h [hunk 1]
```diff
@@ -86,6 +86,22 @@ class QueryUtils final {
         return Status::Error(folly::stringPrintf("Invalid property %s", prop.name_.c_str()));
     }
 
+    static Status collectPropsInValue(RowReader* reader,
+                                      const std::vector<PropContext>* props,
+                                      nebula::List& list) {
+        for (const auto& prop : *props) {
+            if (prop.returned_ && prop.propInKeyType_ == PropContext::PropInKeyType::NONE) {
+                VLOG(2) << "Collect prop " << prop.name_;
+                auto value = QueryUtils::readValue(reader, prop.name_, prop.field_);
```
Comment (bright-starry-sky) [inline src/storage/exec/QueryUtils.h:95]: Do we need to determine the default values?

Comment (critical27) [reply src/storage/exec/QueryUtils.h:95]: Yeah, default values has been handled in `readValue`

Comment (bright-starry-sky) [reply src/storage/exec/QueryUtils.h:95]: > Yeah, default values has been handled in `readValue`

Got, excellent.

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Review by liuyu85cn — APPROVED** [review]
LGTM

**Pull request merged**